packages/harpguru-core/src/components/menu-of-scales/menu-of-scales.tsx

```diff
@@ -9,35 +9,39 @@ import { Menu } from '../menu'
 import { MenuProps } from '../../types'
 import { colors, getSizes } from '../../styles'
 
-import { getTogglesForDispatcher } from './utils'
-import { useScalesTitles, useScalesItemsz } from './hooks'
+import { getTogglesForDispatcher, getToggledActiveQuizDegrees } from './utils'
+import {
+  useScaleTitle,
+  useScaleItems,
+  useQuizQuestionTitle,
+  useQuizQuestionItems,
+} from './hooks'
 import { useImmediatelyFlushToggles } from './hooks'
 
 export const MenuOfScales = (menuProps: MenuProps): React.ReactElement => {
-  const { useScalesTitle, useChordsTitle } = useScalesTitles()
-  const { useScalesItems, useChordsItems } = useScalesItemsz()
   useImmediatelyFlushToggles({
     isMenuStashed: menuProps.isMenuStashed,
   })
-  const itemTapHandler = useCallback(useDispatch(getTogglesForDispatcher), [
-    useDispatch,
-    getTogglesForDispatcher,
-  ])
-
-  const useScalesTitleMemo = useCallback(() => useScalesTitle(useGlobal), [
-    useGlobal,
-  ])
-  const useChordsTitleMemo = useCallback(() => useChordsTitle(useGlobal), [
+  const scaleItemTapHandler = useCallback(
+    useDispatch(getTogglesForDispatcher),
+    [useDispatch, getTogglesForDispatcher]
+  )
+  const useScalesTitleMemo = useCallback(() => useScaleTitle(useGlobal), [
     useGlobal,
   ])
-
   const useScalesItemsMemo = useCallback(
-    () => useScalesItems(useGlobal, itemTapHandler),
-    [useGlobal, itemTapHandler]
+    () => useScaleItems(useGlobal, scaleItemTapHandler),
+    [useGlobal, scaleItemTapHandler]
+  )
+
+  const useQuizQuestionTitleMemo = useCallback(() => useQuizQuestionTitle(), [])
+  const quizQuestionTapHandler = useCallback(
+    useDispatch(getToggledActiveQuizDegrees),
```

Comment:
`getToggledActiveQuizDegrees` should be named `getToggledActiveQuizDegreesForDispatcher` or something similar.

`getTogglesForDispatcher` could probably also do with renaming so that these functions specific purposes are better identified.

Reply:
Done